src/Calc.php

```diff
@@ -22,8 +22,24 @@ public function sayHello(DateTimeImmutable $date): void
         echo 'Hello, ' . $date->format('j. n. Y');
     }
 
+    public $displayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaaayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaa;
+
+    public function sayaHello(DateTimeImutable $date): void
+    {
+        echo 'Hello, ' . $date->format('j. n. Y');
+    }
+
     public function aaa()
     {
         return in_array(1, [1, 2, 3]);
     }
+
+    public function shouryuken()
+    {
+        if (true) {
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
If condition is always true.